Storage/SQL/RemoteTabsTable.swift

```diff
@@ -20,6 +20,7 @@ class RemoteClientsTable<T>: GenericTable<RemoteClient> {
             "type TEXT",
             "formfactor TEXT",
             "os TEXT",
+            "fxaDeviceId TEXT",
```

Comment:
Pretty sure we'll need a DB migration to add this row for existing clients or else bad things will happen. @rnewman would probably know the best way of handling the migration.